docs/source/writing-tests.md

```diff
@@ -305,6 +305,9 @@ There are several global variables you can use to introspect on Bats tests:
 - `$BATS_TEST_NUMBER` is the (1-based) index of the current test case in the test file.
 - `$BATS_SUITE_TEST_NUMBER` is the (1-based) index of the current test case in the test suite (over all files).
 - `$BATS_TMPDIR` is the location to a directory that may be used to store temporary files.
+- `$BATS_RUN_TMPDIR` is the location to the temporary directory used by bats to
```

Comment:
I propose ([here](https://github.com/bats-core/bats-core/pull/410/files#r575897751) as well) we move internal variables to a separate section to minimize folks using them without breaking existing logic.